app/src/main/java/co/touchlab/kmmbridgekickstartandroid/ui/Composables.kt

```diff
@@ -55,11 +55,11 @@ fun MainScreen(
     MainScreenContent(
         dogsState = dogsState,
         onRefresh = { viewModel.refreshBreeds() },
-        onSuccess = { data -> BreedAnalytics.displayingBreeds(data.size) },
-        onError = { exception -> BreedAnalytics.displayingError(exception) },
+        onSuccess = { data -> viewModel.breedAnalytics.displayingBreeds(data.size) },
```

Comment:
I notice that the iOS version of this sample passes the viewmodel _and_ the analytics as separate arguments to the equivalent screen.  On this side the `viewModel.breedAnalytics` chain breaks the _law of demeter_ but it has the benefit of not having to pass the `breedAnalytics` as an argument to the screen (my nitpick with the iOS sample).  You might be able to get the best of all worlds by having the viewModel expose a method (in both Android and iOS samples) to take the `breedAnalytics` out of the construction, and avoid reaching inside the viewModel and breaking the _law of demeter_.